### ISSUE TITLE
Fix Codacy unused variable

### ISSUE THREAD

**Issue by DanRStevens** [lairworks/nas2d-core#172]
Codacy has detected a variable which is assigned, but never used:
[Cadacy Issue: Unused variable](https://app.codacy.com/app/ldicker83/nas2d-core/issues?&filters=W3siaWQiOiJDYXRlZ29yeSIsInZhbHVlcyI6WyJDb2RlU3R5bGUiXX1d)

The code in question is:
```cpp
void OGL_Renderer::drawImageRotated(Image& image, float x, float y, float degrees, int r, int g, int b, int a, float scale)
{
	// ...

	float _deg = degrees;
	if (_deg < 0.0f)
	{
		_deg += 360.0f;
	}

	glRotatef(degrees, 0.0f, 0.0f, 1.0f);

	// ...
}
```

It looks like perhaps `glRotatef(_deg, 0.0f, 0.0f, 1.0f);` was intended.

I should point out though, there is a deeper problem here. This method can only correct by 1 rotation about the circle in a given direction. It doesn't correct for multiple rotations, nor does it handle both directions. For something like that you'd need `degrees % 360.0f`.

Still deeper, the more rotations you are correcting by, the more the error can grow. For large enough floating point values, the delta between adjacent values can actually be larger than 360 degrees, making the correction quite useless.

Further, this correction is not applied back to the source. If it was applied back to the source, keeping the number in a small range, the minimum delta values, and hence the error would also be controlled to a small range.

In summary, I think this angle correction is being done in the wrong place. Such correction makes more sense in the caller, with this method assuming the angle is already in a certain range.

**Comment by cugone** [comment]
Quick note: `% (mod operator)` does not work for non-integral types. Use `std::fmod` instead.

**Comment by DanRStevens** [comment]
I somehow suspected as much. Thank you for looking up the function name:
[`std::fmod`](https://en.cppreference.com/w/cpp/numeric/math/fmod)

It's found in `<cmath>`.

**Comment by ldicker83** [comment]
Honestly, there's no need for any math operations here.

I believe the code is vestigial and I just forgot to remove it. I've been meaning to play with it a bit and do some basic code refactoring.
